lib/internal/crypto/util.js

```diff
@@ -308,6 +309,17 @@ function bigIntArrayToUnsignedInt(input) {
   return result;
 }
 
+function bigIntArrayToBigInt(input) {
```

Comment:
Suggestion, feel free to ignore: For consistency with the previous function, maybe call this `bigIntArrayToUnsignedBigInt`? On the other hand, we will probably never have a use case that requires signed integers.